$NetBSD$

Adjust pkgconfig installation location to lib, not share.

Sent upstream via email 20190427.

--- CMakeLists.txt.orig	2019-04-26 16:07:59.000000000 +0000
+++ CMakeLists.txt
@@ -108,12 +108,12 @@ install(FILES mosquitto.conf aclfile.exa
 # ========================================
 
 configure_file(libmosquitto.pc.in libmosquitto.pc @ONLY)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libmosquitto.pc" DESTINATION "${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libmosquitto.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 configure_file(libmosquittopp.pc.in libmosquittopp.pc @ONLY)
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libmosquittopp.pc" DESTINATION "${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/libmosquittopp.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 
 # ========================================
 # Testing
 # ========================================
 enable_testing()
-add_test("test" make -C ${mosquitto_SOURCE_DIR}/test test)
+add_test("test" gmake -C ${mosquitto_SOURCE_DIR}/test test)
